servers/src/main/java/tachyon/worker/block/evictor/EvictorType.java

```diff
@@ -30,6 +30,13 @@
    */
   LRU,
   /**
+   * Evict old blocks in certain StorageDir by LRU. The main difference between PartialLRU and
+   * LRU is that LRU choose old blocks among several StorageDirs until one StorageDir
+   * satisfies the request space, but PartialLRU select one StorageDir first and evict old
+   * blocks in certain StorageDir by LRU
+   */
+  PARTIAL_LRU,
```

Comment:
@Yours-shupeng yeah, maybe we can share codes between `PartialLRUEvictor` and `LRUEvictor`, if we let `PartialLRUEvictor extends LRUEvictor`, then after selecting the `StorageDir`, we can just call `super` in `freeSpace`.  then PartialLRUEvictor will have feature like "cascading eviction"(https://github.com/amplab/tachyon/pull/1079)  once LRUEvictor has it. What do you think about?